product/views/ProvisionCloudTemplates.yaml

```diff
@@ -0,0 +1,104 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Images
+
+# Menu name
+name: Images
+
+# Main DB table report is based on
+db: TemplateCloud
+
+# Columns to fetch from the main table
+cols:
+- name
+- allocated_disk_storage
+- display_type
+- display_operating_system
+- display_platform
+- cpu_total_cores
+- display_memory
+- v_total_snapshots
+- display_tenant
+- display_deprecated
+
+
+# Included tables (joined, has_one, has_many) and columns
+include:
+  ext_management_system:
+    columns:
+    - name
+
+
+# Included tables and columns for query performance
+include_for_find:
+  :compliances: {}
```

Comment:
There are probably no compliances for templates either - see https://github.com/ManageIQ/manageiq-ui-classic/pull/4958 please :)